docs/howto/build-run.md

```diff
@@ -51,7 +51,7 @@ details to worry about.
 Before starting, install these dependencies if you don't have them:
 * [Git](https://git-scm.com/)
 * [Node.js](https://nodejs.org/en/download/package-manager/): use the
-  **latest 12.x** version, not 14.x or later
+  **latest 16.x** version, or a later LTS version
 * [Yarn](https://yarnpkg.com/en/docs/install), latest stable version
```

Comment:
Does this mean the latest Yarn 1.x (classic) version, by the way?

Reply:
It does. I guess it'd be good to say that explicitly, though it looks like that URL continues to be about the classic versions.

Also I guess at some point we should look at migrating to the "modern" versions. For a while there it looked like a complex migration for something that wouldn't ultimately work for our form of deployment, and it seemed like the developers didn't even really understand why everything hadn't moved to their new model. But it looks like they now promise:
https://yarnpkg.com/getting-started/migration#step-by-step
that the new model isn't required, and that Yarn v2+ should be an improvement even while continuing to use `node_modules`.

Reply:
Noted now: https://github.com/zulip/zulip-mobile/commit/75f20bf2e00ecf181906f693b90b9884c283b9c5